app/poros/event.rb

```diff
@@ -15,18 +17,22 @@ def initialize(data)
     @activity = data[:attributes][:activity]
     @friend_name = data[:meta][:friend_name]
     @friend_role = data[:meta][:friend_role]
+    @gym_name = data[:meta][:gym_name]
+    @yelp_gym_id = data[:meta][:yelp_gym_id]
   end
 
   # TODO: Update method to handle different timezones
   # Returns the date in the format of 'Wednesday, Sep 22, 2021 9:41pm'
   def format_date
-    DateTime.strptime(date_time, '%Y-%m-%dT%H:%M:%S').strftime('%A, %b %d, %Y%l:%M%P')
+    DateTime.strptime(date_time,
+                      '%Y-%m-%dT%H:%M:%S').strftime('%A, %b %d, %Y%l:%M%P')
   end
 
   # TODO: Update method to handle different timezones
-  # Returns the date in the format of 'Wed 09/22/21 9:41pm'
+  # Returns the date in the format of 'Wed 09/22/21- 9:41pm'
   def format_date_short
-    DateTime.strptime(date_time, '%Y-%m-%dT%H:%M:%S').strftime('%a %m/%d/%y%l:%M%P')
+    DateTime.strptime(date_time,
+                      '%Y-%m-%dT%H:%M:%S').strftime('%a %m/%d/%y-%l:%M%P')
   end
```

Comment:
Let's consider storing the raw date in `date_time` and moving these into a module as you had done with the `format_city_state_zip`.

Reply:
Roger that, will update accordingly